instrumentation/opentelemetry-instrumentation-asyncpg/src/opentelemetry/instrumentation/asyncpg/__init__.py

```diff
@@ -133,6 +133,7 @@ async def _do_execute(self, func, instance, args, kwargs):
         exception = None
         params = getattr(instance, "_params", {})
         name = args[0] if args[0] else params.get("database", "postgresql")
+        name = name.split()[0]
```

Comment:
Maybe wrap this in `try...except IndexError...`?

Reply:
good idea! i'll use `name = ""` in except block, since this may happen for both "" and " " (`\s*` in general).